File: client/src/home/borrowing.ts
export type Borrowing = {
  id: number;
  end: Date;
  start: Date;

  bookId: number;
  bookItemId: number;
  userId: number;
};

export type CreateBorrowingDto = {
  bookId: number;
  bookItemId: number;
  start: Date;
  end: Date;
};

File: server/Borrowings/Borrowings/Models/Borrowing.cs
namespace Borrowings.Models;

public class Borrowing
{
    public int Id { get; set; }
    public DateTime Start { get; set; }
    public DateTime End { get; set; }
    public int BookId { get; set; }
    public int BookItemId { get; set; }
    public int UserId { get; set; }
}
